theme/ckeditor5-upload/imageuploadprogress.css

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* Infinite progress bar default width. */
+:root {
+	--ck-image-upload-progress-line-width: 30px;
+}
+
+figure.image {
+	/* Showing animation. */
+	&.ck-appear {
+		animation: fadeIn 700ms;
+	}
+
+	/* Infinite progress bar on top while image is read. */
+	&.ck-infinite-progress::before {
+		width: var(--ck-image-upload-progress-line-width);
+		height: 2px;
+		background: rgba(0, 0, 0, 0.1);
```

Comment:
Would be nice to make variable here `--ck-image-upload-progress-line-background`